packages/graphiql/src/types.tsx

```diff
@@ -0,0 +1 @@
+export type Maybe<T> = T | null;

```

Comment:
do we want `undefined` here as well?

Reply:
This is kind of a subtle point but i think we should keep `undefined` out of here. `null` is still a zero-value whereas `undefined` is explicitly "no value".